start-site/src/main/resources/application.yml

```diff
@@ -142,6 +142,10 @@ initializr:
         mappings:
           - compatibilityRange: "[2.1.0.RELEASE,2.6.0-M1)"
             version: 14.7.5
+          - compatibilityRange: "2.5.x-SNAPSHOT"
+            version: 21.0.8
+          - compatibilityRange: "2.6.x-SNAPSHOT"
```

Comment:
What is the intent here. Offer this version as of Spring Boot 2.6? If so, it should use the same format as above, something like `[2.6.0-M1,2.7.0-M1)`.

Is that version of the Vaadin Spring integration built and tested against Spring Boot 2.6.x?

Reply:
Not quite. The intent is to ship Vaadin v21x with Spring Boot SNAPSHOT only, while Spring Boot stable versions (i.e. 2.5.7 and 2.6.0) would come with Vaadin v14 which is the current LTS (I realize now that the `compatibilityRange` should be further adjusted so that Vaadin 14 would work with 2.6.0). 

The motivation is to get people to try Spring Boot SNAPSHOT + Vaadin vLatest combo so that we can catch any possible issues as early as possible.

Reply:
@tarekoraby that doesn't sound right to me.

Let's assume we merge what you described. `2.6.0` is using the LTS version and `2.6.1-SNAPSHOT` is using the non-LTS version (going 14.x to 21.x!). We're releasing `2.6.1` today. When that's done, `2.6.1` is using the LTS version again.

I am all for letting people try out the next version when they choose to use the SNAPSHOT, but going from 14.x to 21.x isn't reasonable IMO.

Reply:
I see you're point (and it's a valid one). Vaadin LTS still awaits an upcoming fix that should make it compatible Spring Boot 2.6.x. Once that arrives, let's make Vaadin LTS available for all the currently selectable Spring Boot versions (which would mean `compatibilityRange: "[2.1.0.RELEASE,2.6.2-SNAPSHOT]"` if I'm not mistaken.

The fix to Vaadin LTS should arrive next week at the latest. Should I close this PR for now and open a new one then?